crates/delta/src/routes/servers/roles_edit.rs

```diff
@@ -45,22 +45,15 @@ pub async fn edit(
             name,
             colour,
             hoist,
-            rank,
             remove,
+            // rank is deprecated
+            ..
         } = data;
 
-        // Prevent us from moving a role above other roles
-        if let Some(rank) = &rank {
-            if rank <= &member_rank {
-                return Err(create_error!(NotElevated));
-            }
-        }
-
         let partial = PartialRole {
             name,
             colour,
             hoist,
-            rank,
```

Comment:
We should leave this accessible for those using revite
Those using new clients to manage their servers won't use this anyways

Reply:
we're breaking revite with this one ///...